prototypes/prototype.adoc

```diff
@@ -0,0 +1,44 @@
+---
+authors: Han Solo <han@example.com>, Frodo Baggins <frodo@example.com>
+state: predraft
+---
+= RFD <NUMBER> <Title>
+:author: Han Solo <han@example.com>; Frodo Baggins <frodo@example.com>
```

Comment:
Yes, the front matter is being parsed by the chat bot, and I think by 'make check'. I think I'd prefer we just use the asciidoc attributes, and drop the jekyll front-matter for .adoc RFDs.